resources/views/admin.blade.php

```diff
@@ -144,12 +166,65 @@
                                                 <path d="M14.5 3a1 1 0 0 1-1 1H13v9a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V4h-.5a1 1 0 0 1-1-1V2a1 1 0 0 1 1-1H6a1 1 0 0 1 1-1h2a1 1 0 0 1 1 1h3.5a1 1 0 0 1 1 1zM4.118 4 4 4.059V13a1 1 0 0 0 1 1h6a1 1 0 0 0 1-1V4.059L11.882 4zM2.5 3h11V2h-11z"/>
                                             </svg>
                                         </button> `
-                            }
+                            },
+                            width: '6%',
+                        },
+                        {
+                            data: 'company.name',
+                            title:'Company' ,
+                            className: 'column-company' ,
+                            render: DataTable.render.ellipsis( 26 )
+
+                        },
+                        {
+                            data: 'company.branch',
+                            title:'Branch' ,
+                            className: 'column-branch',
+                            render: DataTable.render.ellipsis( 20 )
+
+                        },
+                        {
+                            data: 'name' ,
+                            title: 'Name' ,
+                            className: 'column-name',
+                            render: DataTable.render.ellipsis( 26 )
+                        },
+                        {
+                            data: 'position' ,
+                            title:'Position' ,
+                            className: 'column-position',
```

Comment:
please remove the `className` attributes which are not necessary

Reply:
removed please review again @Raza9798 #60